PL2/PL2-parser/src/main/java/nl/tudelft/pl2016gr2/parser/controller/AnnotationReader.java

```diff
@@ -0,0 +1,211 @@
+package nl.tudelft.pl2016gr2.parser.controller;
+
+import nl.tudelft.pl2016gr2.model.Annotation;
+import org.apache.poi.openxml4j.exceptions.InvalidFormatException;
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.ss.usermodel.Workbook;
+import org.apache.poi.ss.usermodel.WorkbookFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+@SuppressWarnings("checkstyle:AbbreviationAsWordInName")
+public class AnnotationReader {
```

Comment:
Could you add class javadoc which describe what the class does (and preferably also what `@SuppressWarnings("checkstyle:AbbreviationAsWordInName")` does and why it is needed).